docs/pages/switch.js

```diff
@@ -57,6 +57,196 @@ export default function DocsPage({ generatedDocGen }: {| generatedDocGen: DocGen
           />
         </MainSection.Subsection>
       </MainSection>
+      <MainSection name="Best practices">
+        <MainSection.Subsection columns={2}>
```

Comment:
Be sure to wrap every 2 cards in their own subsection (see IconButton for example)

Reply:
Done!